lib/actions/FrontendAction.js

```diff
@@ -59,6 +70,32 @@ class FrontendAction {
         break
       }
     }
+
+    process.on('SIGINT', () => {
+      themeConfigWatcher.stop(() => {
+        logger.info('SDK connection closed')
+        process.exit(0)
```

Comment:
The frontend SDK is showing an end screen after SIGINT. This is interrupted by this `process.exit(0)` Please remove it